front-page.php

```diff
@@ -35,73 +41,100 @@
 
 		?>
 		<div class="homepage homepage__container">
-		<div class="homepage__hero">
-			<div class="homepage__hero__background">
-			</div>
-			<div class="row">
-				<div class="col-md-5 homepage__hero__splash">
-					<div class="homepage__hero__image">
-						<img src="<?php echo $fieldValues->hero_image['url'] ?>" alt="<?php echo $fieldValues->hero_image['alt'] ?>">
-					</div>
-				</div>
-				<div class="col-md-4 col-md-offset-1">
-					<div class="homepage__content">
-						<h1 class="homepage__hero__title title title--main"><?php echo $fieldValues->hero_title; ?></h1>
-						<p class="homepage__hero__subtitle subtitle"><?php echo $fieldValues->hero_subtitle ?></p>
-						<a href="<?php echo (is_user_logged_in() ? esc_attr($fieldValues->hero_cta_existing) : esc_attr($fieldValues->hero_cta_new)) ?>"class="btn btn--dark btn--small homepage__hero__cta"><?php echo __($fieldValues->hero_cta_text) ?></a>
-					</div>
+			<div class="homepage__hero">
+				<div class="homepage__hero__background">
 				</div>
-				</div>
-			</div>
-			<?php if($hasEvents): ?>
-			<div class="homepage__events">
-				<div class="homepage__events__background"></div>
-				<div class="row homepage__events__meta">
-					<div class="col-md-6 col-sm-12">
-						<h2 class="subheader homepage__events__subheader"><?php echo __($fieldValues->featured_events_title)?></h2>
+				<div class="row">
+					<div class="col-md-5 homepage__hero__splash">
+						<div class="homepage__hero__image">
+							<img src="<?php echo $fieldValues->hero_image['url'] ?>" alt="<?php echo $fieldValues->hero_image['alt'] ?>">
+						</div>
 					</div>
-					<div class="col-md-6 col-sm-12 homepage__events__cta">
-						<a href="/events" class="btn btn--small btn--dark"><?php echo __($fieldValues->featured_events_cta_text); ?></a>
+					<div class="col-md-4 col-md-offset-1">
+						<div class="homepage__content">
+							<h1 class="homepage__hero__title title title--main"><?php echo $fieldValues->hero_title; ?></h1>
+							<p class="homepage__hero__subtitle subtitle"><?php echo $fieldValues->hero_subtitle ?></p>
+							<a href="<?php echo (is_user_logged_in() ? esc_attr($fieldValues->hero_cta_existing) : esc_attr($fieldValues->hero_cta_new)) ?>"class="btn btn--dark btn--small homepage__hero__cta"><?php echo __($fieldValues->hero_cta_text) ?></a>
```

Comment:
This doesn't need to be wrapped in __() function.  This should only be used for static text